scripts/embed_duckdb_ext/main.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // DuckDB extensions Rill depends on
-var extensions = []string{"json", "icu", "parquet", "httpfs", "sqlite_scanner", "motherduck"}
+var extensions = []string{"json", "icu", "parquet", "httpfs", "aws", "sqlite_scanner", "motherduck"}
```

Comment:
I know the size is just ~7.5MB but given the use case is not going to be common. I am wondering if we should skip `aws`. Worried about progressively increasing binary size.

Reply:
Makes sense, we can add it back if/when we move sources to use native credentials